elements/pfelement/src/pfelement.js

```diff
@@ -230,10 +230,10 @@ class PFElement extends HTMLElement {
   }
 
   /**
-   * Returns an array with all the slot with the provided name defined in the light DOM.
-   * If no value is provided (i.e., `this.getSlot()`), it returns all unassigned slots.
+   * Given a slot name, returns elements assigned to the slot.
```

Comment:
Please add back the array reference somewhere in here as that is significant since it is using a spread operator to specifically return an array.